openmetadata-spec/src/main/resources/json/schema/configuration/workflowSettings.json

```diff
@@ -50,10 +50,10 @@
       "description": "Used to set up the Workflow Executor Settings."
     },
     "historyCleanUpConfiguration": {
-      "$ref":  "#/definitions/historyCleanUpConfiguration",
+      "$ref": "#/definitions/historyCleanUpConfiguration",
       "description": "Used to set up the History CleanUp Settings."
     }
   },
-  "required": ["allowedClassification", "validityPeriod"],
+  "required": [],
```

Comment:
Is this json change intended?

Reply:
yes, the properties in required, are not present in the json